Makefile

```diff
@@ -160,7 +160,17 @@ test-integration: ## Run integration tests
 	go test -v -tags=integration ./test/integration/...
 
 .PHONY: test-e2e
-test-e2e: docker-build docker-push $(ENVSUBST) $(KUBECTL) $(GINKGO) ## Run e2e tests
+test-e2e: $(ENVSUBST) $(KUBECTL) $(GINKGO) ## Run e2e tests
+	PULL_POLICY=IfNotPresent $(MAKE) docker-build docker-push
```

Comment:
we were running docker build twice, this might also help shave off a few minutes on e2e time